packages/protocol/contracts/common/Accounts.sol

```diff
@@ -85,6 +85,17 @@ contract Accounts is
     setRegistry(registryAddress);
   }
 
+  /**
+   * @notice Sets initialized == true on implementation contracts
+   * @param test Set to true to skip implementation initialization
+   */
+  constructor(bool test) public InitializableV2(test) {}
+
+  modifier isSenderRegistered {
+    require(isAccount(msg.sender), "Sender is not a registered account");
```

Comment:
We often use the prefix `only` for descriptive modifier names
Additionally, we need to parameterize this because `msg.sender` isn't always the account we are checking

```suggestion
  modifier onlyRegisteredAccount(address account) {
    require(isAccount(account), "Account is not registered");
```